client/on_the_record/js/components/asset_history.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import AssetDetail from '../../../lib/js/react/components/asset_detail';
+
+import Spinner from '../../../lib/js/react/components/spinner';
+
+
+export default function AssetHistory(props) {
```

Comment:
Or just a component that is purely displaying things and a component that is handling functionality.

Since it's some sort of a list you want to render, it shouldn't be to difficult, I guess.